crypto/evp/evp_fetch.c

```diff
@@ -167,7 +193,9 @@ void *evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
         method = ossl_method_construct(libctx, operation_id, name,
                                        properties, 0 /* !force_cache */,
                                        &mcm, &mcmdata);
-        ossl_method_store_cache_set(store, id, properties, method);
+        nameid = ossl_namemap_number(namemap, name);
+        methid = method_id(operation_id, nameid);
```

Comment:
This could fail

Reply:
Right, I should check that `method != NULL`.  'cause if there's a method, we know that `method_id` did work